test/node-api/test_uv_threadpool_size/node-options.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const common = require('../../common');
+const assert = require('assert');
+const path = require('path');
+const { spawnSync } = require('child_process');
+
+if (process.config.variables.node_without_node_options) {
+  common.skip('missing NODE_OPTIONS support');
+}
+
+const uvThreadPoolPath = '../../fixtures/dotenv/uv-threadpool.env';
+
+// Should update UV_THREADPOOL_SIZE
+const filePath = path.join(__dirname, `./build/${common.buildType}/test_uv_threadpool_size`);
+const code = `
+   const { test } = require('${filePath}');
+   const size = parseInt(process.env.UV_THREADPOOL_SIZE, 10);
+   require('assert').strictEqual(size, 5);
+   test(size);
```

Comment:
This is the second problem as it hangs in all cases when it is not 4 (actually, if it is smaller than 4 it will reach an assertion failing). I was able to get it to pass by changing the value in `uv-threadpool.env` to 4. I didn't dig deep into the addon itself to know if this is correct or not, but I see that the other test in this folder, also calls `test(4)`.